acquia_lift_profiles/js/acquia_lift_profiles.js

```diff
@@ -232,12 +198,20 @@ var _tcwq = _tcwq || [];
             'trackingId': trackingId
           }, settings.acquia_lift_profiles.pageContext, pageFieldValues);
           _tcwq.push(['onLoad', segmentCallback]);
-          _tcaq.push(['captureView', 'Content View', pageInfo]);
 
-          if(settings.acquia_lift_profiles.hasOwnProperty('identity')) {
-            pushCaptureIdentity(settings.acquia_lift_profiles.identity, settings.acquia_lift_profiles.identityType);
+          // Capture view.
+          var captureViewEvent = ['captureView', 'Content View', pageInfo];
+
+          // Capture identity in addition, if applicable.
+          if(!identityCaptured && settings.acquia_lift_profiles.hasOwnProperty('identity')) {
```

Comment:
Sharp observation Lisa ... 

"captureIdentity" flag is set in Acquia Lift Profiles page and is the switch only for whether or not to capture user_login and user_register. The variable name is misleading as it doesn't control the query string parameters capture identity here.